src/Library/demos/Drop Down/main.blp

```diff
@@ -0,0 +1,73 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Drop Down");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+    valign: center;
+
+    Grid  {
+      orientation: horizontal;
+      halign: center;
+      valign: center;
+      column-spacing: 10;
+
+      Label {
+        halign: start;
+        label: _("Selection:");
+      }
+
+      DropDown drop_down {
+        halign: end;
+        model: StringList {
+          strings ["Item 1", "Item 2", "Item 3", "Item 4"]
+        };
+      }
+    }
+
+    Label {
+        halign: start;
+        label: _("Advanced Drop Down");
+        margin-top: 24;
+        margin-bottom: 24;
+        styles["title-1"]
+      }
+
+     Grid  {
+      orientation: horizontal;
+      halign: center;
+      valign: center;
+      column-spacing: 10;
+
+      Label {
+        halign: start;
+        label: _("Selection:");
+      }
+
+      DropDown advanced_drop_down {
```

Comment:
I think this one should have a label that says "With search" or similar rather than "Advanced"